strings/roundstart_hints.txt

```diff
@@ -133,4 +133,11 @@ Certain items have unique suicide options. Feel free to experiment with them if
 While holding an object, you can press C (Z on TG Controls) to Use/Interact with it. This can either open the crafting menu when holding a sheet of metal, activate a welding tool or make you eat a burger.
 Most PDAs start with a writing utensil which can be ejected to access it! Other things fit in the pen slot as well!
 Your character isn't you. Building up a wall in between the two is important for your mental health.
-If a hole in the ocean floor is just one tile wide you can jump over it by sprinting!
\ No newline at end of file
+If a hole in the ocean floor is just one tile wide you can jump over it by sprinting!
+Don't trust staff assistants! They're not good people.
+If you've grabbed someone with an empty hand, you can pin them by clicking a nearby tile! Stronger grabs and more exhausted people means faster pins.
+Some objects can be used to grab people. Strangle them with a cable coil, or put a knife to their neck so they bleed while being strangled.
+Many items do more damage thrown than used in melee. Throw a chair at the traitor chasing you down with a revolver -- or at the traitor fleeing your toolbox of justice!
+You can only throw a standing person with an aggressive grab, but if someone is knocked down, you can throw them with a passive grab!
+Getting thrown into a vending machine can be quite painful! Watch where you fistfight.
+Sliding into objects on the ground causes you to kick them in that direction.
```

Comment:
I think this could also benefit from a short explanation of how to slide in the first place.